src/app/components/shared/events-list/events-list.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { NotificationService } from '../../../utility/notification.service';
+
+import { EventAPI } from '../../../api';
+
+@Component({
+  selector: 'app-events-list',
+  templateUrl: './events-list.html',
+})
+export class EventsList implements OnInit {
+  @Input() projectId: any = {};
+  @Input() status: any;
+  events: any = [];
+  options: any = {};
+
+  constructor(private eventAPI: EventAPI,
+              private notifyService: NotificationService) {
+    this.options = {
+      group: 'normal-group',
+      onUpdate: (event: any) => {
```

Comment:
This one is duplicated with https://github.com/activebridge/radar-web/pull/17/files#diff-a1830b50513bb8e021af292e986923d2R21 .
Please create function to handle it.
E.g:
```
onUpdate: this.updateEventHandler

onAdd: this.updateEventHandler

updateEventHandler = (event:any) => {
  this.updateEvent(event.item.dataset.eventId, {status: this.status.key.toLowerCase(), position: event.newIndex + 1});
}
```